src/test_wasm_simd.py

```diff
@@ -0,0 +1,127 @@
+#! /usr/bin/env python
+
+import os
+import re
+import subprocess
+import sys
+import tempfile
+
+wasm_simd_test_cases = [
+  # Oper Arity ArgType     ReturnType   Opcode   KnownFailure
+  ('add',  2, 'int32x4',   'int32x4',   'add',   False),
```

Comment:
My unfiltered thoughts here are:

1. Ooh, aligned columns are nice
1. Hm, `True`/`False` are unfortunate literals to have. `True` means "should fail" in this context, which isn't necessarily intuitive. `False` also doesn't map nicely to "should fail". An enum would be nice, but, python.
1. It looks like the return types are always the same as the arg types? That probably isn't always true but happens to be for this set of ops.
1. I feel like having a data class here would make any given line more immediately readable. Something like `TestCase(operation='add', arity=2, arg_type='int32x4')`, and `TestCase(operation='min', arity=2, arg_type='float64x2', opcode='fmin', expected_fail=True)`. Then internal to that constructor we can default opcode to operation when not set, return type defaults to arg type, and expected fail defaults to False. This fixes the known failure boolean mapping.
1. Having the keyword arguments and the constructor adds line length though. If that takes things over 80 characters it makes the whole thing way less skimmable. It also makes aligning the columns meaningfully impossible, again impairing skimmability. And not having the keword args puts us right back in tuple-land.

So... Yeah I don't know what's best here. I'm ok with leaving this as-is, up to you.